### PR TITLE
[WM-2014] Dockstore URL based on env

### PR DESCRIPTION
**Pull request by kpierre13**
__Jira ticket__: https://broadworkbench.atlassian.net/browse/[WM-2014]

- This ticket depends on the changes made in this terra-helm PR: https://github.com/broadinstitute/terra-helmfile/pull/4290

### What

- Adding Dockstore url (supplied by terra-helm) to the coa-app config

### Why

- CBAS-UI wants to navigate to Dockstore given a url root based on the workflows app's environment.

### What to test
- This change was tested in a bee, with an update to the leonardo helm chart. To test, the chart version for leonardo is `2a7c9ef8e0c1` and the terra-helm branch to point to is `wm-2014_pass_dockstore_url_kp`
- Once the Bee has been updated, log in to Terra, crate a new Azure workwspace and spin up the cromwell app. Once in the app, pass in this flag to the console `configOverridesStore.set({ isFindWorkflowEnabled: true, isDockstoreEnabled: true })`, refresh and select the `Find Workflows` button. From there, navigate to the `Dockstore` subheader and click on the `Go to Dockstore` button. Since BEEs use Terra Dev, you should have navigated to the `staging.dockstore.org` url

### Who tested and where

- [ ] This change is covered by automated tests 
  - _NB: Rerun automation tests on this PR by commenting `jenkins retest` or `jenkins multi-test`._
- [x] I validated this change in a BEE on Terra
- [ ] Primary reviewer validated this change 
- [ ] I validated this change in the dev environment 

[WM-2014]: https://broadworkbench.atlassian.net/browse/WM-2014?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by kpierre13** [comment]
jenkins retest

**Comment by kpierre13** [comment]
jenkins retest

**Comment by kpierre13** [comment]
jenkins retest

**Comment by kpierre13** [comment]
jenkins retest

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala [hunk 1]
```diff
@@ -548,6 +550,7 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
         raw"cbas.enabled=${config.coaAppConfig.coaServices.contains(Cbas)}",
         raw"cbasUI.enabled=${config.coaAppConfig.coaServices.contains(CbasUI)}",
         raw"cromwell.enabled=${config.coaAppConfig.coaServices.contains(Cromwell)}",
+        raw"dockstore.baseUrl=${config.coaAppConfig.dockstoreBaseUrl}",
```
Comment (LizBaldo) [inline http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala:554]: This is how the COA helm chart Values.yaml file is expecting this baseURL correct? Just checking as I have been bitten by this before 😅

**Review by LizBaldo — APPROVED** [review]
That looks great to me :) I just left a tiny comment to double check the corresponding coa helm chart values to make sure there is a match.

That being said, if you were able to launch a COA app from your BEE pointing to these two PRs then you should be all set 👍

File: http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala (hunk 1 above)
Comment (kpierre13) [reply http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala:554]: Yes! I was getting plenty of errors in my BEE before realizing I my naming was off 😅 but this was tested in a BEE and worked as expected!

**Review by salonishah11 — APPROVED** [review]

**Pull request merged**